packages/salesforcedx-vscode-core/src/telemetry/telemetry.ts

```diff
@@ -62,14 +69,16 @@ export class TelemetryService {
       );
       this.context.subscriptions.push(this.reporter);
     }
+
+    this.setCliTelemetryEnabled(this.isTelemetryEnabled() && !isDevMode);
```

Comment:
Why do we need to consider `!isDevMode` here ?

Reply:
@lcampos So that CLI telemetry is not captured for dev mode.

Reply:
I think that the CLI should be doing that internally, just like we are doing a few lines above. If this is not the case we should open a bug on it. It shouldn't be the consumer's responsibility to determine if it's a dev scenario or not.

Reply:
@lcampos I'll remove that check and open a bug if appropriate.